we_get/core/module.py

```diff
@@ -20,22 +19,19 @@ def http_get_request(self, url):
         """ http_request: create HTTP request.
           @return: data.
         """
-        opener = urllib.request.build_opener()
-        opener.addheaders = [('User-Agent', USER_AGENT), ("Accept", "*/*")]
-        try:
-            res = opener.open(url).read().decode()
-            return res
-        except URLError as err:
+        headers = [('User-Agent', USER_AGENT), ("Accept", "*/*")]
+        res = requests.get(url, headers)
+        if res.status_code == 200:
```

Comment:
use try-except to get `res.text` even on non 200 status code result

but replace `URLError` with `Exception` so url will be printed when any error happen